src/components/Navbar/NavigationDrawer/LanguageContainer.module.scss

```diff
@@ -0,0 +1,67 @@
+@use 'src/styles/utility';
+@use 'src/styles/breakpoints';
+
+.languageContainer {
+  top: 0;
+  height: 100%;
+  position: absolute;
+  inset: 0;
+  display: flex;
+  flex-direction: column;
+  background: var(--color-background-default);
+  transform: translateX(100%);
+  transition: transform var(--transition-regular) ease-in-out;
+  z-index: var(--z-index-default);
+
+  [dir='rtl'] & {
+    transform: translateX(-100%);
+  }
+
+  &.show {
+    transform: translateX(0);
+  }
+}
+
+.languageHeader {
+  display: flex;
+  align-items: center;
+  gap: var(--spacing-small);
+  padding: var(--spacing-medium);
+  border-block-end: 1px solid var(--color-borders-hairline);
+}
+
+.backButton {
+  padding: var(--spacing-xsmall);
```

Comment:
For all new css, we should use pixel values not rem as per agreenment with Talha to make sure what is on Figma is 100% the same pixel values in the code base. We already have pixel values suffixed with `-px` e.g. `--spacing-xxsmall-px` etc. Please apply to entire file.